pmpro-member-directory.php

```diff
@@ -158,3 +158,35 @@ function pmpromd_plugin_row_meta($links, $file) {
 	return $links;
 }
 add_filter('plugin_row_meta', 'pmpromd_plugin_row_meta', 10, 2);
+
+/**
+ * Should we make something clickable. Filters included
+ */
+function pmpromd_make_clickable( $url, $field = false ){
+
+	/**
+	 * Logic follows:
+	 * 1) Do we want to make it an oembed URL, if yes, do it. If not, check pt2
+	 * 2) Do we want to make it clickable. If yes, do it. If not, check pt3
+	 * 3) Just return a string/URL
+	 */
+	if( apply_filters( 'pmpromd_make_clickable_oembed', true, $field ) ) {
+		
+		$url_embed = wp_oembed_get($url);
+		if( !empty( $url_embed ) ){
+			return $url_embed;
+		}
+
+		return $url;
+
+	} else if ( apply_filters( 'pmpromd_make_clickable', false, $field ) ) {
+
+		return make_clickable($url);
```

Comment:
If we output the value exactly as it is (make_clickable just makes links/emails clickable, it doesn't escape content) then we may have an issue here.

Maybe we should use something like `esc_html()` or `wp_kses_post()` here.

```suggestion
		return make_clickable( esc_html( $url ) );
```

```suggestion
		return make_clickable( wp_kses_post( $url ) );
```

Reply:
`wp_kses_post()` seems the most sane option here